code/modules/research/faction_destructibles.dm

```diff
@@ -0,0 +1,46 @@
+//my sincerest apologies for any shitcode, I'm not fixing it
+
+
+/obj/item/faction_tech
+	name = "Encrypted Design Drive"
+	//desc = "A large drive for storing equipment design data for construction in protolathes. One use, and <span class='italics'>heavily</span> encrypted, so you'll have to process it in a destructive analyzer first."
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "blackcube"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+
+/obj/item/faction_tech/nanotrasen
+	name = "NanoTrasen Design Drive"
+
+/obj/item/faction_tech/hardliners
+	name = "Gorlex Hardliners Design Drive"
+
+/obj/item/faction_tech/gorlex_republic
+	name = "New Gorlex Republic Design Drive"
+
+/obj/item/faction_tech/gorlex_marauders
+	name = "Gorlex Marauders Design Drive"
+
+/obj/item/faction_tech/cybersun
+	name = "Cybersun Design Drive"
+
+/obj/item/faction_tech/inteq
+	name = "Inteq Design Drive"
+
+/obj/item/faction_tech/clip
+	name = "CLIP Design Drive"
+
+/* do we actually need this one?
```

Comment:
yes for same reasons posted above; primarily used for frontiersmen ships/possibly indy ships